docs/getting-started.md

```diff
@@ -4,15 +4,55 @@ title: Getting Started
 hide_title: false
 ---
 
-This serves as a guide for new Cumulus users to deploy and learn how to use Cumulus.
+[Overview](#overview) | [Quick Tutorials](#quick-tutorials) | [Helpful Tips](#helpful-tips)
+
+## Overview
+
+This serves as a guide for new Cumulus users to deploy and learn how to use Cumulus. Here you will learn what you need in order to complete any prerequisites, what Cumulus is and how it works, and how to successfully navigate and deploy a Cumulus environment.
+
+### What is Cumulus
+
+Cumulus is a cloud-based data ingest, archive, distribution and management prototype for NASA's future Earth Science data streams.
+
+### Who uses Cumulus
+
+Data integrators/developers and operators use Cumulus for their daily work functions.
+
+### Cumulus Roles
+
+As a developer, integrator, or operator, you will need to set up your environments to work in Cumulus. The following docs can get you started in your role specific activities.
+
+* [Developers](https://github.com/nasa/cumulus)
+* [Integrators](integrator-guide/about-int-guide)
+* [Operators](operator-docs/about-operator-docs)
+
+### What is a Cumulus Data Type
+
+In Cumulus, we have the following types of data that you can create and manage:
+
+* Collections
+* Granules
+* Providers
+* Rules
+* Workflows
+* Executions
+* Reports
+
+For details on how to create or manage data types go to [Data Management Types](configuration/data-management-types).
+
+---
+
+## Quick Tutorials
+
+### Deployment & Configuration
 
 Cumulus is deployed to an AWS account, so you must have access to deploy resources to an AWS account to get started.
 
-## 1. Deploy Cumulus and Cumulus dashboard to AWS
+### 1. Deploy Cumulus and Cumulus Dashboard to AWS
 
 Follow the [deployment instructions](deployment/deployment-readme) to deploy Cumulus to your AWS account.
 
-## 2. Configure and run the HelloWorld workflow
+### 2. Configure and Run the HelloWorld Workflow
 
 If you have deployed using the [cumulus-template-deploy repository](https://github.com/nasa/cumulus-template-deploy), you have a `HelloWorld` workflow deployed to your Cumulus backend.
 
```

Comment:
Can you do me a favor please and change the link on line 61 to `data-cookbooks/about-cookbooks#setup`